x-pack/functionbeat/tests/system/config/functionbeat.yml.j2

```diff
@@ -78,3 +32,21 @@ output:
 
     # Number of rotated log files to keep. Oldest files will be deleted first.
     #keepfiles: 7
+
+#============================= Elastic Cloud ==================================
+
+# These settings simplify using filebeat with the Elastic Cloud (https://cloud.elastic.co/).
+
+# The cloud.id setting overwrites the `output.elasticsearch.hosts` and
+# `setup.kibana.host` options.
+# You can find the `cloud.id` in the Elastic Cloud web UI.
+cloud.id: {{cloud_id}}
+
+# The cloud.auth setting overwrites the `output.elasticsearch.username` and
+# `output.elasticsearch.password` settings. The format is `<user>:<pass>`.
+cloud.auth: {{cloud_auth}}
```

Comment:
Do we want to make these optional, so tests could also be run against a non ES Cloud instance?